tools/cloud-build/daily-tests/ansible_playbooks/tasks/create_deployment_directory.yml

```diff
@@ -0,0 +1,52 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+- name: Assert variables are defined
+  ansible.builtin.assert:
+    that:
+    - blueprint_yaml is defined
+    - project is defined
+    - workspace is defined
+    - deployment_name is defined
+    - test_name is defined
+
+- name: Create cli flag for extra deployment variables
+  ansible.builtin.set_fact:
+    deployment_vars_str: "--vars {{ cli_deployment_vars.items() | map('join', '=') | join(',') }}"
+  when: cli_deployment_vars is defined
```

Comment:
Add `and cli_deployment_vars is mapping` (after double-checking my syntax below)

https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_tests.html#type-tests

Reply:
Added.